source/numbers.rst

```diff
@@ -68,8 +69,8 @@ behaviour we expect from basic arithmentic.
     >>> 5 - 6
     -1
 
-The behaviour is to compute and return the result as a new object with the same
-type.
+The behaviour is to compute and return the result as a new number object with the same
+type. In the above example `5 + 4` equals the 9 which is of type `int`.
```

Comment:
I think this is potentially confusing because if you can add together and int and a float and you get a float and, if we do opt to use python 3, you can divide two ints and get a float. I'd just leave these sentences out.